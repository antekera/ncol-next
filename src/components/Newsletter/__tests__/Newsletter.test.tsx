import { render } from '@testing-library/react'
import { Newsletter } from '..'

describe('Newsletter', () => {
  test('should match snapshots', () => {
    const { container } = render(<Newsletter />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
