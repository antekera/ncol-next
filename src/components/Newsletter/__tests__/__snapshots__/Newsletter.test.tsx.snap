// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Newsletter should match snapshots 1`] = `
<div
  class="newsletter"
>
  <form
    action="javascript:throw new Error('A React form was unexpectedly submitted. If you called form.submit() manually, consider using form.requestSubmit() instead. If you\\'re trying to use event.stopPropagation() in a submit event handler, consider also calling event.preventDefault().')"
  >
    <label
      class="newsletter-label"
      for="email-input"
    >
      Suscríbete a nuestro boletín
    </label>
    <p
      class="newsletter-description"
    >
      Recibe grátis las noticias más destacadas en tu correo.
    </p>
    <div
      class="newsletter-input-wrapper"
    >
      <div
        class="newsletter-input-container"
      >
        <input
          autocapitalize="off"
          autocorrect="off"
          class="newsletter-input"
          id="email-input"
          name="email"
          placeholder="tu.correo@mail.com"
          required=""
          type="email"
          value=""
        />
      </div>
      <div
        class="newsletter-button-wrapper"
      >
        <button
          class="button focus:shadow-outline"
        >
          <svg
            class="lucide lucide-mail"
            fill="none"
            height="20"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <rect
              height="16"
              rx="2"
              width="20"
              x="2"
              y="4"
            />
            <path
              d="m22 7-8.97 5.7a1.94 1.94 0 0 1-2.06 0L2 7"
            />
          </svg>
          Suscribirme
        </button>
      </div>
    </div>
  </form>
</div>
`;
